### ISSUE TITLE
Problems exporting and importing playlists etc. 

### ISSUE THREAD

**Issue by hagueyno1** [TeamNewPipe/NewPipe#5869]
### Checklist

- [x] I am using the latest version - 0.20.11 
- [ ] I checked, but didn't find any duplicates (open OR closed) of this issue in the repo. 
- [ ] I have read the contribution guidelines given at https://github.com/TeamNewPipe/NewPipe/blob/HEAD/.github/CONTRIBUTING.md.
- [ ] This issue contains only one bug. I will open one issue for every bug report I want to file.

### Actual behaviour

Hi, 
I've spoken to a dev regarding exporting all my playlists etc from one phone to another which he told me what I knew, I've tried using the back to back part too and it does pop up transfer data but it never comes up on my new phone, I have tried other ways, my old phone is storing the data ready but new phone won't find it, I do only have one sim card but I'm also using the hot-spot and tried Bluetooth also but I still can't get it to mine across from a Samsung galaxy s7 to a Samsung j4 plus, the data just doesn't appear for me to get it on the j4. 
Am I missing something or is this a bug? 
I've even been through each file looking for it on the j4 to no avail. 
It would be great if they're were a really easy way to do it or if you can explain exactly how to do it please, I have so many songs to do it one by one, although if needs be ill have to one day. 

### Expected behavior

I expect it to be relatively easy like your dev said it would be, maybe just put the phones back to back and it's done. 

### Logs

B from 

### Device info
From Samsung galaxy s7 to Samsung j4 plus. 
 - Android version/Custom ROM version:
 - Device model:
Samsung galaxy s7 running android 8.0,
Samsung j4 plus running android 9.0.

**Comment by hagueyno1** [comment]
I'm not what people would call tech savvy although I'm usually good with phones, it's just I have around 2-3000 tracks on my galaxy s4 and could really do with a way to transfer them in one go, I have in the past done then all one by one which took forever and I will again if needs be although I have A considerable amount more traks than I used to, so I'm begging could someone please explain in layman's terms how I do it, if you have read my original post I honestly have tried admittedly I may have got a step wrong, I've managed it before but with less than 60 tracks, I think I got lucky.
Please help me. Thanks, James Hague.

**Comment by mhmdanas** [comment]
Just to clear up a potential misunderstanding here: NewPipe only handles the data exporting parts; it doesn't know what you do with the file. Thus, this is not an issue with NewPipe, but with the file transfer methods you are using. I'll be closing this issue because of that, but you can still make comments to communicate with me or others.

You could try something like https://snapdrop.net; it will download the export to your downloads directory I believe. Note that both the sending and recipient devices have to be on the same network.

If you're still having trouble, please make a comment.
